engine/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/EmailPayloadAggregatorFactory.java

```diff
@@ -30,8 +29,6 @@ public static AbstractEmailPayloadAggregator by(EmailAggregationKey aggregationK
                 switch (application.toLowerCase()) { // TODO Remove toLowerCase if possible
                     case ANSIBLE:
                         return new AnsibleEmailAggregator();
-                    case RHOSAK:
```

Comment:
Unrelated, are we still onboarding `ansible` ? It looks like it's part of the `application-services`  bundle aswell.